cpu/native/include/periph_conf.h

```diff
@@ -49,7 +49,8 @@ extern "C" {
  * @name Timer peripheral configuration
  * @{
  */
-#define TIMER_NUMOF        (1U)
+#define TIMER_NUMOF            (1U)
+#define TIMER_CHANNEL_NUMOF    (1U)
```

Comment:
```suggestion
#define TIMER_CHANNEL_NUMOF    (1U)     /**< number of timer channels */
```

I'm well aware that the name is pretty self-documenting, but the CI insists on having this documented. The alternative would be to add this to the list of exceptions, but that is IMHO more effort that just documenting the obvious.